### ISSUE TITLE
f_rt_all

### ISSUE THREAD

**Issue by noahmthomas-nmt** [milleratotago/ASR_Miller_Schwarz_R#1]
I think that your density functions rely on a f_rt_all() that may have been retooled to f_rt(). This seems to have happened in your version history. The arguments f_rt() also seems to be more complicated compared to f_rt_all().

**Comment by milleratotago** [comment]
You are right. The retooling involved making the model more general to allow different values of sigma_C for cases where A versus B finished first. Unfortunately the asr_predictions routines were not brought up to date with that retooling.  I think they are now.
